tests/gold_tests/network_error/epipe.test.py

```diff
@@ -0,0 +1,45 @@
+'''
+'''
+#  Licensed to the Apache Software Foundation (ASF) under one
+#  or more contributor license agreements.  See the NOTICE file
+#  distributed with this work for additional information
+#  regarding copyright ownership.  The ASF licenses this file
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from ports import get_port
+import sys
+
+Test.Summary = '''
+Test that Trafficserver starts with default configurations.
+'''
+
+epiper = Test.Processes.Process("epiper")
+epiper.Setup.CopyAs('epiper.py', Test.RunDirectory)
+origin_port = get_port(epiper, "Port")
+epiper.Command = f"{sys.executable} epiper.py {origin_port}"
+epiper.Ready = When.PortOpen(origin_port)
+
+ats = Test.MakeATSProcess("ats", select_ports=True)
```

Comment:
A small thing, but most of our tests just use `ts` for the process name. I suggest keeping with that convention.